src/main.rs

```diff
@@ -287,6 +296,26 @@ fn settings() -> Result<Settings, Failure> {
         |path| Some(PathBuf::from(path)),
     );
 
+    // Read the config file path.
+    let default_output_path = toastfile_path.parent().unwrap().to_path_buf();
```

Comment:
Generally, any usage of `unwrap()` (except in tests) must be justified by a comment ([example](https://github.com/stepchowfun/toast/blob/611f7a70e6a2329e445740a3442b189b52d142a1/src/docker.rs#L342-L344)).

In this case, I don't think it is actually justified. If `toastfile_path` happens to be `/` (e.g., because the user ran `toast -f /`), this will panic, right? (Panics are generally unacceptable in this codebase.)

So I think we need to handle this failure scenario and return some kind of error.